test/fixtures/parser/bigquery/select_where_less_than.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+  *
+FROM
+  table_a
+WHERE
+  -- Tests that '<' is parsed correctly. (Since some dialects use angle
```

Comment:
@alanmcruickshank: I found a new parsing error using the BigQuery dialect. This query parses correctly in `master`. I strongly suspect that somehow the new angle bracket handling is causing this. This file fails to parse with the following error message:

```
Couldn't find closing bracket for opening bracket.
```

Any ideas?

Reply:
Crud - I think I was overoptimistic with something else. Thanks for adding the test case. I've got a good idea about how to resolve it.

Reply:
I imagine that's resolved now? I just checked out the PR and tried to parse that file and that worked